metadata/pkg/query-translator/QueryTranslator.go

```diff
@@ -0,0 +1,178 @@
+package querytranslator
+
+import (
+	"github.com/opensds/multi-cloud/metadata/pkg/constants"
+	pb "github.com/opensds/multi-cloud/metadata/proto"
+	log "github.com/sirupsen/logrus"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+func Translate(in *pb.ListMetadataRequest) []bson.D {
+	var aggOperations []bson.D
+	aggOperations = constructAggOperationForBackendLevel(in, aggOperations)
```

Comment:
can you please add example of aggregation as comment.

Reply:
![image](https://user-images.githubusercontent.com/48655639/214242324-8de5bb01-a6ec-469e-9e0e-548039fb0efe.png)
done